common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/WorksQuery.scala

```diff
@@ -0,0 +1,57 @@
+package uk.ac.wellcome.elasticsearch
+
+import com.sksamuel.elastic4s.ElasticDsl._
+import com.sksamuel.elastic4s.requests.common.Operator.{AND, OR}
+import com.sksamuel.elastic4s.requests.searches.queries.BoolQuery
+import com.sksamuel.elastic4s.requests.searches.queries.matches.MultiMatchQueryBuilderType.BEST_FIELDS
+import com.sksamuel.elastic4s.requests.searches.queries.matches.{
+  FieldWithOptionalBoost,
+  MultiMatchQuery
+}
+import uk.ac.wellcome.elasticsearch.WorksAnalysis.whitespaceAnalyzer
+
+case object WorksMultiMatcher {
```

Comment:
Has this been removed in the API project? Might be being stupid but can't see it

Reply:
Indeed, fixed.